src/libraries/System.IO/tests/Net5CompatTests/System.IO.Net5Compat.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <!-- Windows is currently the only OS for which we provide a new strategy, so we test the Net5Compat only for Windows -->
     <TargetFrameworks>$(NetCoreAppCurrent)-windows</TargetFrameworks>
+    <IgnoreForCI Condition="'$(TargetOS)' != 'windows'">true</IgnoreForCI>
```

Comment:
> Windows is currently the only OS for which we provide a new strategy

I think that no longer is the case since we have [`Net5CompatFileStreamStrategy.Unix.cs`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/Strategies/Net5CompatFileStreamStrategy.Unix.cs). 
cc @adamsitnik

Reply:
> I think that no longer is the case.

That is true. `System.IO.Net5Compat.Tests.csproj` should be executed for every OS (to ensure we have not broken anything between .NET 5 and 6)

Reply:
Ok, then we need to change the TargetFrameworks to NetCoreAppCurrent :)
Will do that and check why it fails on Android.

Reply:
I've sent a PR to address that: https://github.com/dotnet/runtime/pull/58611

the tests are passing locally on my Ubuntu machine.

Reply:
Ok thanks, I just verified they're passing after this PR on Android too. They were actually failing for the same reason regarding F_SETLK64 on Android arm but I got confused since this .csproj just includes the test source files so the namespaces are different.